packages/rocketchat-livechat/.app/client/views/register.js

```diff
@@ -27,7 +27,13 @@ Template.register.helpers({
 	},
 	showEmailFieldRegisterForm() {
 		return Livechat.emailFieldRegistrationForm;
-	}	
+	},
+	getName() {
+		return Template.instance().guestName.get();
```

Comment:
you can use `Livechat.guestName` directly here, instead of using the reactive var